### ISSUE TITLE
.NET 3.1 installation fails

### ISSUE THREAD

**Issue by shaunm-msft** [MicrosoftDocs/intellicode#312]
When I open vscode from WSL, I get this error:

Downloading the .NET Runtime.
Downloading .NET version(s) 3.1.18 ... Error!
Failed to download .NET 3.1.18:
Command failed: "/home/shaunm/.vscode-server-insiders/extensions/ms-dotnettools.vscode-dotnet-runtime-1.2.0/dist/install scripts/dotnet-install.sh" -InstallDir "/home/shaunm/.vscode-server-insiders/data/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/3.1.18" -Version 3.1.18 -Runtime dotnet
/home/shaunm/.vscode-server-insiders/extensions/ms-dotnettools.vscode-dotnet-runtime-1.2.0/dist/install scripts/dotnet-install.sh: 7: /home/shaunm/.vscode-server-insiders/extensions/ms-dotnettools.vscode-dotnet-runtime-1.2.0/dist/install scripts/dotnet-install.sh: Syntax error: newline unexpected

 Error!
.NET Acquisition Failed: Installation failed: Error: Command failed: "/home/shaunm/.vscode-server-insiders/extensions/ms-dotnettools.vscode-dotnet-runtime-1.2.0/dist/install scripts/dotnet-install.sh" -InstallDir "/home/shaunm/.vscode-server-insiders/data/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/3.1.18" -Version 3.1.18 -Runtime dotnet
/home/shaunm/.vscode-server-insiders/extensions/ms-dotnettools.vscode-dotnet-runtime-1.2.0/dist/install scripts/dotnet-install.sh: 7: /home/shaunm/.vscode-server-insiders/extensions/ms-dotnettools.vscode-dotnet-runtime-1.2.0/dist/install scripts/dotnet-install.sh: Syntax error: newline unexpected

This is coming from .NET Install Tool for Extension Authors which Visual Studio IntelliCode Insiders depends upon.
I cannot uninstall the .NET extension because Intellicode depends on it.

**Product and Version** [VS/VSCode]:
Version: 1.61.0-insider (user setup)
Commit: ec40cbe164c59f3d13b6eff5fc7b0231ec0ab7aa
Date: 2021-09-13T05:15:14.689Z (20 hrs ago)
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043

**IntelliCode Extension Version**: 
Visual Studio IntelliCode Insiders v1.1.4

**Steps to Reproduce / Scenario:**

1. From a WSL terminal: code-insiders .

**Comment by leti367** [comment]
https://github.com/dotnet/vscode-dotnet-runtime/issues/159 is open to track this issue. I will also check if there's any solution to bypass this issue.

**Comment by leti367** [comment]
The new version of '.NET Install Tool for Extension Authors' has been released and it fixes issue. Let me know if you still see the issue(you might need to uninstall '.NET Install Tool for Extension Authors' to get rid of cache issue).

**Comment by shaunm-msft** [comment]
I uninstalled the retail Intellicode and installed IntelliCode Insiders for WSL. I got the same dotnet installation error. Interestingly, Intellicode Insiders thinks it is installed. I can hover and see examples on GitHub, so it must be installed and working. Maybe the error was left over from a previously installed extension; I'm not sure. 

I tried installing on another machine. That machine received no error. So it appears that the error has been fixed.
